README.md

```diff
@@ -46,15 +51,25 @@ else interest_rate = 10%
 
 **1st example:**
 
-You made your first deposit `1000 tokens` and then made an instant withdrawal in or near this block. In this case, your earned interest will be 0 or very small and you will pay a commission about `30 tokens` and get about `970 tokens` back.
+User deposits `1000 tokens` then makes an instant withdrawal in or near this block. In this case, earned interest will be close to 0 with a fee of about `30 tokens`. User will receive `970 tokens` back.
 
 **2nd example:**
 
-You made your first deposit `1000 tokens`. Then you decided to make one more deposit `1000 tokens` after `2 weeks`. As it is the 1st interval the contract will pay you `5%` annual interest `(1000 * 0.05) / 365 * 14 = 1.92` and it will update your deposit date to the current date. Now you have `2001.92 tokens`, and `3 months` later you decided to withdraw all. `3 months >= (1 month + 2 months)` so this is the 3rd interval. In this case, the contract will pay you `7%` annual interest `(2001.92 * 0.07) / 365 * 90 = 34.55` and you will receive `2036.47 tokens`.
+User deposits `1000 tokens`. After `2 weeks`, the user makes a 2nd deposit of `1000 tokens`. This is within the 1st interval, so the contract will pay  `5%` annual interest `(1000 * 0.05) / 365 * 14 = 1.92` and will update the deposit date to the current date. The new balance will be `2001.92 tokens`.
```

Comment:
```suggestion
User deposits `1000 tokens`. After `2 weeks`, the user makes the 2nd deposit of `1000 tokens`. This is within the 1st interval, so the contract will pay  `5%` annual interest `(1000 * 0.05) / 365 * 14 = 1.92` and will update the deposit date to the current date. The new balance will be `2001.92 tokens`.
```
Isn't it?

Reply:
I think `a` is correct here - but if confusing using `the` works too.